pkg/nodejs/prepare/application.go

```diff
@@ -47,7 +47,10 @@ COPY ./{{.PackageDirectory}}/{{.Static}} /u01/application/static
 
 COPY nginx.conf /etc/nginx/nginx.conf
 
-ENV MAIN_JAVASCRIPT_FILE="/u01/application/{{.MainFile}}" IMAGE_BUILD_TIME="{{.ImageBuildTime}}"
+ENV MAIN_JAVASCRIPT_FILE="/u01/application/{{.MainFile}}" \
+    IMAGE_BUILD_TIME="{{.ImageBuildTime}}" \
+    PROXY_PASS_HOST="localhost" \
```

Comment:
Mulig det bare er meg som misforstår her, men er PROXY_PASS_HOST en build time config nå (den bør være runtime)?

Reply:
Glem det. Så den andre pr'en ;)

Reply:
Men, er det fleksibelt nok å kun kunne sette host og port?

Reply:
Hva annet tenker du å sette?

Hvis du tenker på path, så syns jeg ikke den skal være konfigurerbar.

Reply:
Ok. Det var igrunnen hele urlen (med path) jeg tenkte kunne være nyttig å sette. Forøvrig kan det være greit å ha en konvensjon på det. Så lenge det det ikke fører til problemer for dem som skal bruke dette (med eksisterende api) er jeg enig.

Reply:
Min erfaring med proxyer, er at rewrite av path medfører mye flere problemer enn hva de løser. Det blir blant annet veldig vanskelig å linke i responser.

Det å kunne justere "location" kan vi legge inn støtte for om det er et behov. Men støtter konvensjon. Gjør det lettere om vi skal lage mod_security-regler også.